pkg/code-generator/docgen/funcs/template_funcs.go

```diff
@@ -318,8 +318,13 @@ func linkForField(project *model.Project, docsOptions *options.DocsOptions) func
 				linkedFile = filepath.Base(file.GetName())
 				//return "", errors.Errorf("failed to get generated file path for proto %v in list %v", file.GetName(), project.Request.FileToGenerate)
 			}
-			linkedFile = relativeFilename(forFile.GetName(), linkedFile)
 
+			// Don't link to external apis docs, as they will be deleted
+			if strings.HasPrefix(linkedFile, "github.com/solo-io/gloo/projects/gloo/api/external") {
```

Comment:
is there a way to make this non-gloo-specific? e.g. pass in the repo name to the func?

Reply:
good idea, implemented, and wrapped it up in a struct with this issue in mind: https://github.com/k8sgateway/k8sgateway/issues/10349